src/app/event/_components/AdvertisePanel.module.scss

```
@@ -0,0 +1,44 @@
+.container {
+  @include flex-column(0);
+
+  align-items: center;
+  justify-content: space-between;
+  width: 26rem;
+  height: 28rem;
+  overflow: hidden;
+  border-radius: 1.2rem;
+  background: url('../../../../public/images/backconfetti.png'), linear-gradient(to left, #d3e1ff, #fcfcf2);
```

절대경로로 하는 게 더 좋을 것 같아요!
```suggestion
  background: url('/images/backconfetti.png'), linear-gradient(to left, #d3e1ff, #fcfcf2);
```

---

이게 안되더라고용,, 방법 있으면 알려주세용,, ㅠ